src/ol_infrastructure/applications/learn_ai/__main__.py

```diff
@@ -665,19 +616,19 @@
             dest_secret_labels=k8s_global_labels,
             dest_secret_name=db_creds_secret_name,
             labels=k8s_global_labels,
-            mount=learn_ai_db_vault_backend_config.mount_point,
+            mount=ol_app_db_service.app_db_vault_backend_config.mount_point,
             path="creds/app",
             restart_target_kind="Deployment",
             restart_target_name="learn-ai-app",
             templates={
-                "DATABASE_URL": f'postgres://{{{{ get .Secrets "username"}}}}:{{{{ get .Secrets "password" }}}}@{db["address"]}:{learn_ai_db_config.port}/{learn_ai_db_config.db_name}',
+                "DATABASE_URL": f'postgres://{{{{ get .Secrets "username"}}}}:{{{{ get .Secrets "password" }}}}@{db["address"]}:{ol_app_db_service.app_db_config.port}/{ol_app_db_service.app_db_config.db_name}',
             },
             vaultauth=vault_k8s_resources.auth_name,
         ),
         opts=ResourceOptions(
             delete_before_replace=True,
             parent=vault_k8s_resources,
-            depends_on=[learn_ai_db_vault_backend],
+            depends_on=[ol_app_db_service.app_db_vault_backend],
```

Comment:
```suggestion
            depends_on=[ol_app_db_service],
```
Using the vault backend attribute vs. the overarching component is exposing too much detail at this layer of the code.

Reply:
No longer testing against learn.